.env.sample

```diff
@@ -3,9 +3,6 @@
 # as this file is stored as plain text on disk,
 # and is therefore not secure enough.
 
-# Name
-YOUR_NAME=
-
```

Comment:
This change needs to be made upstream, in https://github.com/hedera-dev/hedera-tutorial-demo-base-template/blob/main/.env.sample
and then replicated here

Reply:
done https://github.com/hedera-dev/hedera-tutorial-demo-base-template/pull/1